plaso/formatters/gdrive_synclog.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""Google Drive Sync log event formatter."""
+
+from __future__ import unicode_literals
+
+from plaso.formatters import interface
+from plaso.formatters import manager
+
+
+class GDriveSyncLogFormatter(interface.ConditionalEventFormatter):
+  """Formatter for a Google Drive Sync log file event."""
+
+  DATA_TYPE = 'gdrive_sync:log:line'
+
+  FORMAT_STRING_PIECES = [
+      '[{log_level}',
+      '{pid}',
+      '{thread}',
+      '{source_code}]',
+      '{message}']
+
+  FORMAT_STRING_SHORT_PIECES = ['{message}']
+
+  SOURCE_LONG = 'GDriveSync Log File'
+  SOURCE_SHORT = 'LOG'
+
+  # TODO: do we want a GetMessages method here? Current iteration has some
```

Comment:
GetMessages, is for formatting enumerations and other forms of output strings that can be derived from the event object but do not necessarily need to be stored.

Reply:
Thanks for the context -- removed the TODO for now and we can consider expanding with GetMessages if necessary in the future.